site/Dance/styles.css

```diff
@@ -165,6 +235,18 @@ footer {
 
 }
 
+@media (max-width: 600px) {
+  .interaction-buttons-2 {
+    flex: 1 1 0;
+    flex-wrap: wrap;
+    flex-basis: 0;
+    flex: 1 0 50%;
+    flex-grow: 1;
+    flex-shrink: 0;
```

Comment:
This flex stuff can get tricky. One thing I want to point out is you have a lot of flex styles overwriting each other. You have `flex` defined twice, on lines 240 and 243. Also, lines 240 and 243 are setting flex as shorthand (similar to my border comment above). So, when you do:

`flex: 1 2 3;`

it's the same as doing:

```
flex-grow: 1;
flex-shrink: 2;
flex-basis: 3;
```

Try and clean this up so you don't have stuff overwriting each other. Either go with doing it shorthand and just setting `flex` with 3 values or you can set each value individually.